docs/collections/redis-mode/redis-mode-indexes.md

```diff
@@ -1,26 +1,19 @@
 ---
-title: Document Store Indexes
-sidebar_position: 50
+title: Redis Mode Indexes
```

Comment:
Is Indexes correct english or should we use Indices?

Reply:
Both versions are correct, but "indexes" is simpler to understand and is the version we most often use in technical writing.

Reply:
@diana-macrometa @martin-macrometa +1, we also use "indexes" in the GUI, so I would prefer we stick with that.